checkout/access-checkout-android/src/main/java/com/worldpay/access/checkout/session/request/handlers/VerifiedTokensSessionRequestHandler.kt

```diff
@@ -42,6 +59,9 @@ internal class VerifiedTokensSessionRequestHandler(
         sessionRequestHandlerConfig.getContext().startService(serviceIntent)
     }
 
+    /**
+     * Takes in [CardDetails] and returns a [CardSessionRequest] object.
```

Comment:
Comments like this is not really needed, the signature in this method is quite self-explanatory